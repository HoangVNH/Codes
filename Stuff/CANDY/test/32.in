10
38
VVVVXXDDDDDDDVVVVVXXXXXXXXXXXXDDDVVDDD
50
DVVVVVDDDDDDDXXXXXXXXVVVVXXXXDDDDDDDDXXXDVVVVDDDDD
43
VVVXXXXXVVXXXXXVVVVVXXDDDDXXXXDDDXXXXXVXXXX
29
VVVVVVVVVXXXXVVVVVXXXXXVVVDDD
43
VVVVVVVVDDDXVVVVVVVVVVDDDDDXXXXXXXXDDDDDDDV
38
DDDDDDDVVVXVVVVVXDDVVVVVVVVVVDDDDDDVVV
49
DDDDDDDDDXVVVVVVVVVVXXXXXXVVVVVVDDXXXXVVVVDDDDDDD
37
XXXXXDDDDDVVVVDDDDDDVVVVDDDDDXXXXXVXD
49
VXXXDDDXXXXDDDVVVVVDDDDDDDDXXXXXDDDDDDDDDDXDDDDDD
48
DDXXXVVVVXVVVVVDXVVVVXXVVVVVDDDDVVDDXXXXXXXDDXXX
