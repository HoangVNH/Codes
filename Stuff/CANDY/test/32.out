37
49
41
27
41
37
48
35
48
47
